Gemfile

```diff
@@ -7,6 +7,10 @@ ruby '2.7.4'
 # Rails essentials
 #
 
+# for working with the map
+gem 'leaflet-rails'
+gem 'leaflet-locatecontrol-rails', git: 'https://gitlab.com/kendrikat/leaflet-locatecontrol-rails.git'
+
```

Comment:
These lines have no effect since we use leaflet from JS, not Ruby, and install leaflet and its dependencies in javascript. If the indoor map uses leaflet features in ruby, we still need these, but should coordinate so that we don't import it twice

Reply:
you are right, we removed it for now